src/material/datepicker/calendar-body.ts

```diff
@@ -310,21 +313,24 @@ export class MatCalendarBody<D = any> implements OnChanges, OnDestroy, AfterView
    */
   _focusActiveCell(movePreview = true) {
     this._ngZone.runOutsideAngular(() => {
```

Comment:
I don't think you need `runOutsideAngular` anymore. `afterRender*` callbacks run outside the zone already.